redfish-core/lib/service_root.hpp

```diff
@@ -34,6 +34,34 @@
 namespace redfish
 {
 
+inline void
+    handleACFWindowActive(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    const std::variant<bool>& retVal) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR << "Failed to read ACFWindowActive property";
+            // Default value when panel app is unreachable.
+            asyncResp->res.jsonValue["Oem"]["IBM"]["ACFWindowActive"] = false;
+            return;
+        }
+        const bool* isACFWindowActive = std::get_if<bool>(&retVal);
+        if (isACFWindowActive == nullptr)
+        {
+            BMCWEB_LOG_ERROR << "nullptr for ACFWindowActive";
+            messages::internalError(asyncResp->res);
+            return;
+        }
+        asyncResp->res.jsonValue["Oem"]["IBM"]["ACFWindowActive"] =
+            *isACFWindowActive;
+        },
+        "com.ibm.PanelApp", "/com/ibm/panel_app",
+        "org.freedesktop.DBus.Properties", "Get", "com.ibm.panel",
```

Comment:
All code upstream is using  sdbusplus::asio::getProperty

https://github.com/openbmc/bmcweb/blob/3ccb3adb9a14783f6bef601506de9f8bcae22d51/redfish-core/lib/led.hpp#L39

 I would prefer if we did the same.